ui/src/flux/components/BodyBuilder.tsx

```diff
@@ -9,11 +9,12 @@ import BodyDelete from 'src/flux/components/BodyDelete'
 import {funcNames} from 'src/flux/constants'
 
 import {Service} from 'src/types'
-import {Body, Suggestion} from 'src/types/flux'
+import {Body, Suggestion, Context} from 'src/types/flux'
 
 interface Props {
-  service: Service
   body: Body[]
+  service: Service
+  context: Context
```

Comment:
if you are going to pass context down as props, could you name which context it is as well? maybe `fluxContext`.